### ISSUE TITLE
Logo not being replaced

### ISSUE THREAD

**Issue by StClairAdmin** [Librarika/Issues#418]
I am trying to change my library logo.  I select the file (.png, 200 x 200 pixels), the new logo file name appears, but  the picture of the logo does not change, .  When  I Submit, it says the information has been saved, but the logo does not change. I have closed and re-opened Librarika and still no new logo. I have tried this many times.

Is there a way to delete the existing logo so I can add the new one?

Thanks
Linda Skinner

**Comment by StClairAdmin** [comment]
I took another screen shot of the logo.  The new one worked.

**Comment by MattSQL** [comment]
I'm having this same issue. No error messages. Loading a .png 200 x 200 file just not appearing. Any solutions?

**Comment by StClairAdmin** [comment]
I tried this many, many times, with no success.  Then I re-took the logo screen shot and it up-loaded just fine.

Hope this works for you.

Linda

**Comment by be2n** [comment]
I have a broken image file appearing when I upload my logo via the logo update feature.
I've tried gif, jpeg, png all at 200x200 with all the same result. I'm on a free plan. 

Is there something I'm missing?

**Comment by StClairAdmin** [comment]
Sorry be2n, I don't know what you mean by a 'broken image file'.

**Comment by be2n** [comment]
![screenshot-silverstripe librarika com-2018 09 14-15-50-18](https://user-images.githubusercontent.com/33236568/45528758-eb952e00-b835-11e8-874d-5060585e515c.png)
![screenshot-silverstripe librarika com-2018 09 14-15-49-26](https://user-images.githubusercontent.com/33236568/45528759-ec2dc480-b835-11e8-91fb-e22ff44c573f.png)
@StClairAdmin as captured in the screenshots

**Comment by StClairAdmin** [comment]
Hmmm... I have no idea, but I do know that I just kept trying and trying (probably over a week or two). Someone said it may be a timing (delay) issue, but I never had the patience to wait as long as it seemed to take. As I said in an earlier post, at one point when I recreated the logo file, it worked. I think it was a fluke and that the logo update function just doesn't work consistently.  I know this doesn't help and it's really frustrating, but maybe if you leave it for a few days, recreate the file and try again, it might work. From my experience, it does work ... eventually. I'd be interested in hearing the outcome.

Linda Skinner

p.s. I too use the free version. Maybe it works better in the paid versions.
